lib/google-civic/error_on_retriable_response.rb

```diff
@@ -0,0 +1,76 @@
+module GoogleCivic
+
+  class RetriableError < Exception
+  	attr_reader :env, :response
+
+  	def initialize(env)
+  	  @env = env
+  	end
+  end
+
+  class ErrorOnRetriableResponse < Faraday::Middleware
+
+  	SERVER_ERROR_CODES = (500...600)
+
+  	TEMPORARY_REASONS = Set.new(["concurrentLimitExceeded", "rateLimitExceeded",
+  		 						 "servingLimitExceeded", "userRateLimitExceeded"])
```

Comment:
Yeah, it looks like in any new rb files I made, it defaulted to tab 4 spaces and I missed that.